Rakefile

```diff
@@ -1,5 +1,6 @@
+require 'puppet_litmus/rake_tasks' if Bundler.rubygems.find_name('puppet_litmus').any?
```

Comment:
this entire file, gives me feels. But it is needed :(

Reply:
Agreed, but I didn't see another clean way to do it.  Particularly as Puppet-Strings exposes public Rake Tasks too.  Didn't want to pollute those.

That said, I tried to make my additions "modular" so when Litmus does support this stuff it'll be easier to port (I hope!)